_docs/_latest/setup/upgrade.md

```diff
@@ -398,6 +398,46 @@ Example command: `gcloud compute instances reset forseti-server-vm-70ce82f --zon
 1. Configuration file `forseti_conf_server.yaml` updates:  
 Forseti is updated to be usable on a non organization resource.
 
+{% capture upgrading_2_4_0_to_2_5_0 %}
+
+Starting v2.5,Forseti Inventory Crawler will be integrated with the Cloud 
+Asset Inventory (CAI) service by default if the user is deploying Forseti. CAI
```

Comment:
I would remove `by default if the user is deploying Forseti`.  Seems too verbose.